geo/src/algorithm/line_measures/mod.rs

```diff
@@ -0,0 +1,16 @@
+//! Line measurements like [`Bearing`] and [`Distance`] for various metric spaces like [`Haversine`], [`Geodesic`], and [`Rhumb`].
```

Comment:
Could we mention `geo`'s "main" metric space (Euclidean) here before the others?

Reply:
Good catch!